docs/.vuepress/styles/index.styl

```diff
@@ -0,0 +1,8 @@
+.home .hero .action-button
+	background-color: #217FF9
```

Comment:
```suggestion
	background-color: #354eff
```

Reply:
I'd say make it lighter then, this color is still pretty bright, and it doesnt really go well with the blue/purple that we have in the logo. A lighter blue makes it a bit easier on the eyes instead:
<img width="600" alt="screen shot 2019-01-16 at 21 21 12" src="https://user-images.githubusercontent.com/17054057/51276333-b57add00-19d4-11e9-999e-12ee79d44fd5.png">